src/azure-cli/azure/cli/command_modules/keyvault/_params.py

```diff
@@ -124,22 +125,21 @@ class CLIJsonWebKeyOperation(str, Enum):
         c.argument('vnet_name', help='Name of a virtual network.', validator=validate_subnet)
 
     with self.argument_context('keyvault private-endpoint-connection', min_api='2018-02-14') as c:
-        c.argument('approval_description', help='Comments for the approval.')
+        c.argument('description', help='Comments for the approval/rejection.')
```

Comment:
You can consider my implementation in storage using string format for the help message to customize it in specific scenario.
 https://github.com/Azure/azure-cli/blob/3e796e8e1f70e0c75027279916cf1f886f3e8d12/src/azure-cli/azure/cli/command_modules/storage/_params.py#L222

Reply:
> You can consider my implementation in storage using string format for the help message to customize it in specific scenario.
> https://github.com/Azure/azure-cli/blob/3e796e8e1f70e0c75027279916cf1f886f3e8d12/src/azure-cli/azure/cli/command_modules/storage/_params.py#L222

Good point.

Reply:
@Juliehzl Another concern, why all your operations have "description" parameter here? Should it only belongs to "approve" and "reject"?